ui/js/graphs.js

```diff
@@ -738,6 +741,15 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
   var testArray = [];
   var series = [];
   $scope.addTestData = function () {
+    if ($scope.addedTestList.length + numTestsDisplayed > 6) {
+      var a = window.confirm('WARNING: Displaying more than 6 graphs at the same time is not recommended. Do it anyway?');
+      if (a == true) {
+        addTestToGraph();
+      }
+    } else addTestToGraph();
```

Comment:
Oh, forgot to mention, you should wrap the else statement with braces for clarity here.

i.e.

    else {
      addTestToGraph();
    }